api-docs.js

```diff
@@ -0,0 +1,176 @@
+
+const apiDoc = {
+  openapi: '3.0.0',
+  servers: [
+    { url: "http://localhost:8000" }
+  ],
+  info: {
+    title: 'Headless Hathor Wallet API',
+    description: 'This wallet is fully controlled through an HTTP API.',
+    version: '1.0.0',
```

Comment:
Nitpick: I'd try to use `0.1.0` for anything that just started.

Reply:
I agree, actually all our repositories have version `0.X.Y`

Reply:
I agree. Fixed. Thanks!